page-object-code-gen/src/main/java/com/kaspresso/components/pageobjectcodegen/KotlinCodeGenerator.kt

```diff
@@ -0,0 +1,33 @@
+package com.kaspresso.components.pageobjectcodegen
+
+abstract class KotlinCodeGenerator(val elements: List<BaseView>, val filePackage: String) : Generator {
+    override fun generate(writer: TextWriter) {
+        with(writer) {
+            if (filePackage.isNotEmpty()) {
+                append("package $filePackage", 2)
+            }
+            createImports(elements).forEach {
+                append(it)
+            }
+            nextLine()
+        }
+    }
+
+    private fun createImports(screenElements: List<BaseView>): List<String> {
+        val importsList = mutableSetOf("import com.screen.common.KScreen", "import ${screenElements.first().packages}.R")
+
+        for (element in screenElements) {
+            when (element.viewType) {
+                "ImageView" -> importsList.add("import io.guthub.kakaocup.kakao.image.KImageView")
```

Comment:
nit: would be better to move string literals into constants or even evaluate package dynamically `KImageView::class.qualifiedName`